package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "agent-base": "^6.0.2",
     "async-retry": "^1.3.3",
+    "crc-32": "^1.2.2",
```

Comment:
This is one swiftly drive-by comment, to my understanding the npm module `crc-32` only been used during tests, but somehow got listed direct into `dependencies` instead of `devDependencies`?